package.json

```diff
@@ -139,25 +140,21 @@
     }
   },
   "lint-staged": {
-    "linters": {
-      "src/**/*.css": [
-        "stylelint --fix"
-      ],
-      "src/**/*.{js,jsx}": [
-        "prettier --write",
-        "eslint --no-ignore"
-      ]
-    },
-    "ignore": [
-      "src/**/*.test.{js,jsx}",
-      "src/**/assets/*.css"
+    "src/**/*": [
+      "prettier --write"
+    ],
+    "src/**/*.{js,jsx}": [
+      "eslint --no-ignore"
```

Comment:
I think you want a `--fix` on this one.

Reply:
you know... I was under some assumption that our eslint would override some of the prettier settings, but I didn't realize that our eslintrc.js is actually extending the eslint prettier plugin. Good catch, I'll fix this.